ci-operator/step-registry/ipi/conf/aws/blackholenetwork/blackhole_vpc_yaml.md

```diff
@@ -173,6 +173,25 @@ Resources:
     Properties:
       SubnetId: !Ref PrivateSubnet3
       RouteTableId: !Ref PrivateRouteTable3
+  EC2Endpoint:
```

Comment:
based on docs https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-vpcendpoint.html

`SecurityGroupIds` is required when using Interface type, also `SubnetIds` is required for that type.

The security group needs to allow ingress from entire VPC.

Reply:
also without PrivateDnsEnabled set to true the machine-api will not automatically use this private link interface.

Reply:
Once we get the proxy instance fixed, we should be able to see these issues in the periodics.

Reply:
#11750 fixed the proxy instance setup (SSH access is still broken, but is not critical).  I've pushed a639ef9b97 -> ae4da132f3 with a stab at your recommended changes, and have updated the live, long-running blackhole VPC stacks to match.  But I wouldn't be surprised if I've still got the new security group wrong.  I guess we'll see how rehearsals are working out...